views/index.ejs

```diff
@@ -1,10 +1,44 @@
 <% include partials/header %>
+<div class="FlagImgContainer"></div>
```

Comment:
better name, more generic like `HeroImgContainer`, this photo wont always be a flag.